systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -330,9 +330,9 @@ public static boolean verifyPodDynamicConfiguration(String kafkaPodNamePrefix, S
 
         Map<String, ConfigModel> configs = KafkaUtils.readConfigModel(kafkaVersion);
 
-        LOGGER.info("This is configs {}", configs.toString());
+        LOGGER.debug("Kafka config {}", configs.toString());
```

Comment:
I guess that it's good to know which dynamic configuration will be tested. The sentence is good, just change it back to the `info` level.